File: src/main/java/ru/hollowhorizon/mastertech/api/item/ItemBase.java
package ru.hollowhorizon.mastertech.api.item;

import net.minecraft.item.Item;
import ru.hollowhorizon.mastertech.MasterTech;
import ru.hollowhorizon.mastertech.api.model.IModeled;

public class ItemBase extends Item implements IModeled {
    public ItemBase() {
        setCreativeTab(MasterTech.tab);
    }

    @Override
    public void registerModels() {
        MasterTech.proxy.getRender(this, 0, "inventory");
    }
}

File: src/main/java/ru/hollowhorizon/mastertech/api/item/ItemBase.kt
package ru.hollowhorizon.mastertech.api.item

import net.minecraft.item.Item
import ru.hollowhorizon.mastertech.MasterTech
import ru.hollowhorizon.mastertech.api.model.IItemModeledByDefault
import ru.hollowhorizon.mastertech.api.model.IModeled

open class ItemBase : Item(), IItemModeledByDefault {
    init {
        creativeTab = MasterTech.tab
    }
}
